app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -70,42 +72,52 @@ def number_of_deposits(user)
       ActiveFedora::Base.where(DepositSearchBuilder.depositor_field => user.user_key).count
     end
 
-    def link_to_facet(field, field_string)
-      path = search_action_path(search_state.add_facet_params_and_redirect(field_string, field))
-      link_to(field, path)
+    # @param item [Object]
+    # @param field [String]
+    # @return [ActiveSupport::SafeBuffer] the html_safe link
+    def link_to_facet(item, field)
+      path = search_action_path(search_state.add_facet_params_and_redirect(field, item))
+      link_to(item, path)
     end
 
-    # @param values [Array] The values to display
-    # @param solr_field [String] The name of the solr field to link to without its suffix (:facetable)
-    # @param empty_message [String] ('No value entered') The message to display if no values are passed in.
-    # @param separator [String] (', ') The value to join with.
-    def link_to_facet_list(values, solr_field, empty_message = "No value entered", separator = ", ")
+    # @param values [Array{String}] strings to display
+    # @param solr_field [String] name of the solr field to link to, without its suffix (:facetable)
+    # @param empty_message [String] message to display if no values are passed in
+    # @param separator [String] value to join with
+    # @return [ActiveSupport::SafeBuffer] the html_safe link
+    def link_to_facet_list(values, solr_field, empty_message = "No value entered".html_safe, separator = ", ")
       return empty_message if values.blank?
       facet_field = Solrizer.solr_name(solr_field, :facetable)
       safe_join(values.map { |item| link_to_facet(item, facet_field) }, separator)
     end
 
+    # @param name [String] field name
+    # @param value [String] field value
+    # @param label [String] defaults to value
+    # @param facet_hash [Hash] first argument to Blacklight::SearchState.new
+    # @return [ActiveSupport::SafeBuffer] the html_safe link
+    # @see Blacklight::SearchState#initialize
     def link_to_field(name, value, label = nil, facet_hash = {})
       label ||= value
       params = { search_field: 'advanced', name => "\"#{value}\"" }
       state = search_state_with_facets(params, facet_hash)
       link_to(label, main_app.search_catalog_path(state))
     end
 
-    def index_field_link(options = {})
-      field_name = options[:config][:field_name]
+    # A Blacklight helper_method
+    # @param options [Hash] Blacklight sends :document, :field, :config, :value and whatever else was in options
+    # @option options [Hash] :config including {:field_name => "my_name"}
+    # @option options [Array{String}] :value
+    # @return [ActiveSupport::SafeBuffer] the html_safe link
+    def index_field_link(options)
```

Comment:
Because you were going to throw an exception in the next line anyway.  In practice, it was already required.